cfme/tests/test_appliance.py

```diff
@@ -373,6 +373,25 @@ def test_appliance_exec_scripts():
     pass
 
 
+@pytest.mark.tier(2)
+def test_appliance_contains_ansible_modules(appliance):
+    """
+    check that there are ansible modules included in the appliance.
+
+    Polarion:
+        assignee: jhenner
+        casecomponent: Appliance
+        caseimportance: high
+        initialEstimate: 1/6h
+        startsin: 5.9.2
+    Bugzilla:
+        1678130
```

Comment:
This test is for automating RFR right? Please add `automates` meta if possible.

Reply:
Oh, I forgot. THanks